### DISCUSSION TITLE
请问pag能否使用gpu渲染

### DISCUSSION THREAD

**Discussion by jackTJT** [Tencent/libpag#2165] [General] [answered]
请问pag能否使用gpu渲染，我看之前有说可以替换EGL库，但是替换后还是渲染一次占用了300-400%多的cpu，替换后程序确实占用了显卡资源，但是渲染好像还是用的cpu资源

**Comment by kevingpqi123** [comment, marked as answer]
如果你指的是服务器端，默认采用的是 CPU 的渲染模式，因为 GPU 的服务器成本太高了
如果需要适配，参考下这个帖子：
https://github.com/Tencent/libpag/discussions/1319
